contrib/vscode/init.sh

```diff
@@ -5,6 +5,15 @@ die () {
 	exit 1
 }
 
+SYSTEM_NAME="$(awk -F= '/^NAME/{print $2}' /etc/os-release)"
```

Comment:
That's a neat find! However, on other systems, that file may not even exist, and then the script shows a nasty error message that may leave the user puzzled. Also, we can avoid using `awk` altogether by doing this:

```sh
if grep '^NAME="Ubuntu"' /etc/os-release >/dev/null 2>&1
then
...
fi
```

Also, please note that Git's shell script conventions dictate the the `then` keyword lives on its own, separate line.